### PR TITLE
fix(tags): parse timezone information from a tag

### PR DESCRIPTION
**Pull request by shcheklein**
We don't parse timezones, means we are not getting right dates in Studio. Also Studio tests produce a lot of warning about "naive" date times.

Not critical, but better to fix in advance.

### CODE REVIEW

File: gto/tag.py [hunk 1]
```diff
@@ -124,17 +124,18 @@ class Tag(BaseModel):
...
-    created_at: datetime
+    created_at: datetime.datetime
     tag: GitTag
 
     class Config:
         arbitrary_types_allowed = True
 
 
 def parse_tag(tag: GitTag):
+    tz = datetime.timezone(datetime.timedelta(minutes=int(tag.tag_time_offset)))
```
Comment (shcheklein) [inline gto/tag.py:135]: C: Offset come in a form of `540`, `-180`, etc. @pmrowla can we confirm that it is always the case? and that we are always getting `tag_time_offset` in the GitTag?

Comment (pmrowla) [reply gto/tag.py:135]: The timezone offset should be in seconds, looks like one of the scmrepo backends is returning minutes (which is an scmrepo bug)

File: gto/tag.py [hunk 2]
```diff
@@ -124,17 +124,18 @@ class Tag(BaseModel):
...
     return Tag(
         tag=tag,
-        created_at=datetime.fromtimestamp(tag.tag_time),
+        created_at=datetime.datetime.fromtimestamp(tag.tag_time, tz=tz),
```
Comment (pmrowla) [inline gto/tag.py:138]:
```suggestion
        created_at=tag.tag_datetime,
```

File: gto/tag.py (hunk 1 above)
Comment (pmrowla) [reply gto/tag.py:135]: scmrepo 1.4.0 is released with the fix for this, and you can also now use `tag.tag_datetime` to get the timezone-aware python datetime

Comment (shcheklein) [reply gto/tag.py:135]: thanks @pmrowla !

**Comment by pmrowla** [comment]
We should also bump the scmrepo minver to 1.4.0 in setup.py, other than that LGTM

**Pull request merged**